charts/stable/rtsp-to-web/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.0.0
```

Comment:
```suggestion
appVersion: v2.2.0
```

Reply:
I lost these when resolving merge conflicts and re-applied manually.